wordpress-mobile/WordPress-iOS
# Invite People: limit Message to 500 characters

Ref: #15356 

This limits the custom message to 500 characters. 

Tech side note: The `NSLocalizedString`s are kind of scattered throughout, and some are duplicated/very similar. In a follow-up PR, I'll look into extracting them into a `Constants` or something. Something less repetitive anyway. 😄 

To test:

Character limiting:
- Go to Site > People.
- Tap '+' in the nav bar.
- Tap on the `Custom message...` field.
- Type an opus exceeding 500 characters. Or, in `InvitePersonViewController`, change this value to something manageable: `private let messageCharacterLimit = 500`
- Verify once the limit is reached, no more characters are entered.

_Note_: there is an outstanding issue (#10795) where the entire message is not displayed in the invite Message view.

Footer text:
- Verify on both views (the Invite view and the Message entry view) the footer text now references the character limit.

| ![invite](https://user-images.githubusercontent.com/1816888/102547573-a0060600-4076-11eb-9ecc-0ce7de26e7d7.png) | ![message](https://user-images.githubusercontent.com/1816888/102547590-a72d1400-4076-11eb-9125-d78c439eeece.png) |
|--------|-------|

PR submission checklist:
